### ISSUE TITLE
Make sure that this sim works with mobile

### ISSUE THREAD

**Issue by zepumph** [phetsims/gravity-force-lab-basics#119]
I'm unsure about the priority of this, but I know that we don't currently support number pickers in a11y, and that should be solved by https://github.com/phetsims/sun/issues/499, but we should make sure to test this in general. Not sure what the priority is, but it should block publication.

**Comment by terracoda** [comment]
@terracoda is reviewing content changes that rely on Focus and Blur events :-)

**Comment by terracoda** [comment]
@zepumph, I mentioned in https://github.com/phetsims/a11y-research/issues/138#issuecomment-487534714 that I reviewed the design of the `aria-valuetext` and I assigned you a couple of questions to you from the design doc. We can discuss at our next meeting.

And with changing the Mass number spinners to sliders with `aria-roledescription` should mean we shouldn't have any mobile issues with this sim (hopefully).

**Comment by jessegreenberg** [comment]
phetsims/sun#499 got us most of the way there to mobile VO working with NumberPicker. But the values in GFLB are creating problems in mobile VO because the `step` attribute is so much smaller than the mass range (1/1000000000).  https://github.com/phetsims/sun/issues/509 was made for that issue.

**Comment by terracoda** [comment]
@jessegreenberg, I am confused, can't the step be 1, and the range be 1 to 10?

**Comment by jessegreenberg** [comment]
@terracoda for sure, that would also solve the issue. The current implementation of this sim has both the physical model and NumberPicker units in kg so the range goes from 1000000000 - 10000000000. Only the display of the value is in billions of kg. Ideally, alternative input should well for any range and step size, so https://github.com/phetsims/sun/issues/509 was to see if it could be improved in common code.

But a fix was committed in that issue and input is now working well with mobile VO.

**Comment by jessegreenberg** [comment]
I just tested on an iPad Air 2 with VoiceOver and the sim works well. @zepumph @terracoda is there anything remaining for this issue?

**Comment by terracoda** [comment]
@jessegreenberg, perhaps not. We do have some general design changes that we are working on to not rely on focus events. I think these will be handled generally, though, so maybe not effect this issue.

**Comment by zepumph** [comment]
I'm going to unassign for now, but keep the blocking label on it. That will guarantee that we can have one more test either before or during dev testing.

**Comment by terracoda** [comment]
@jessegreenberg and @zepumph, I think we have addressed this issue, so unassigning myself, too.

**Comment by zepumph** [comment]
I will make sure to mention this in the dev testing issue I create today. Closing
